### DISCUSSION TITLE
vte3 catch up

### DISCUSSION THREAD

**Discussion by inoueta** [Homebrew/discussions#3173] [Everyday usage]
Now, the formula of vte3 is the version 0.64.2.
There are newer releases, and 0.66.2 seems stable one.
So, we had better to update vte3, I think.
Does anyone can create a pull request for us?
I don't have private GitHub. Thanks.

**Comment by SMillerDev** [comment]
Please see https://docs.brew.sh/Updating-Software-in-Homebrew, discussions are not the way to request updates

**Comment by Bo98** [comment]
This is already being worked on. It should be merged within a few days.
